card/mod/standard/set/all/rich_html/form.rb

```diff
@@ -19,8 +19,8 @@
   end
 
   view :edit_in_form, cache: :never, perms: :update, tags: :unknown_ok do
-    @form = form_for_multi
-    @within_multi_edit = true
+    reset_form
+    @edit_in_form = true
```

Comment:
I find this nomenclature much clearer.  line 22 was actually just resetting the form, even before.  It _looked_ like it was creating a new kind of form (a "form_for_multi"), but all forms were generated with that method.  That confused me.

Similarly, I think just changing this variable to @edit_in_form makes this easier to hold in your head.

Reply:
I'm not convinced that @edit_in_form is an improvement. There are a lot of methods with `multi` and `single` in the name to distinguish between those two cases. Since an edit view is always a form, edit_in_form doesn't make it clear to me that it belongs to the multi case.

Reply:
yeah, I liked the idea of having the view and variable have the same name.

For now I'll just change it so the test and the variable have the same name and use `@in_multi_card_editor`.  (The only other method that mentions this variable is `#in_multi_card_editor?`, which only checks to see whether the variable is present!)  This will satisfy my need to reduce all the naming noise.